src/client-joo/target_table.ml

```diff
@@ -274,8 +274,19 @@ module Filter = struct
             | Some old, Some `All -> Some `All
             | Some (`Created_after t1), Some (`Created_after t2) ->
               Some (`Created_after (min t1 t2)))
-      | `Not _ ->
-        None (* we reach the limits of this weird logic … *)
+      | `Not something ->
+        begin match something with
+        | `Has_tags _
+        | `Name _ | `Id _
+        | `Status _ -> None
+        | `All -> Some (`Created_after max_float)
```

Comment:
That was the previous behavior which made `(not (all))` be `None` so the UI would show the same result as `(created-after 1970)` (which is in practice is `(all)`).

Now `(not (all))` shows nothing (by hackishly asking for anything created after the end of the universe).